zeebe/stream-platform/src/test/java/io/camunda/zeebe/stream/impl/ProcessingScheduleServiceTest.java

```diff
@@ -67,7 +68,9 @@ void before() {
             lifecycleSupplier,
             () -> testWriter,
             commandCache,
-            actorScheduler.getClock());
+            actorScheduler.getClock(),
+            Duration.ofSeconds(1),
+            ScheduledTaskMetrics.noop());
```

Comment:
:wrench: With Micrometer, we can now also inject our `SimpleMeterRegistry` to effectively test metrics properly. I think a simple test would suffice. I understand this isn't the most important metric, but I would like to encourage us starting, and this seems as good a place to start as any :)

Reply:
The `SimpleMeterRegistry` really lends itself well for this :+1: 
Added a couple of basic tests now.